azure-communication-ui/demo-app/src/calling/java/com/azure/android/communication/ui/callingcompositedemoapp/CallLauncherViewModel.kt

```diff
@@ -16,10 +16,13 @@ import com.azure.android.communication.ui.calling.CallComposite
 import com.azure.android.communication.ui.calling.CallCompositeBuilder
```

Comment:
Please see this commit. I have changed this file to be aligned to latest refactoring. I think you can just copy/past: https://github.com/Azure/communication-ui-library-android/commit/b11da9ca73c14673d48a75ac25799f717ddce730